walker.go

```diff
@@ -18,7 +18,7 @@ type WalkOpt struct {
 	Map             func(*Stat) bool
 }
 
-func Walk(ctx context.Context, p string, opt *WalkOpt, fn filepath.WalkFunc) error {
+func Walk(ctx context.Context, p string, opt *WalkOpt, fn filepath.WalkFunc) (err error) {
```

Comment:
Perhaps good to rename the output variable to prevent confusion

Reply:
updated